Doc/library/sqlite3.rst

```diff
@@ -47,83 +47,104 @@ This document includes four main sections:
       PEP written by Marc-André Lemburg.
 
 
+.. We use the following practises for SQL code:
+.. - UPPERCASE for keywords
+.. - lowercase for schema
+.. - single quotes for string literals
+.. - singular for table names
+.. - if needed, use double quotes for table and column names
```

Comment:
```suggestion
   - UPPERCASE for keywords
   - lowercase for schema
   - single quotes for string literals
   - singular for table names
   - if needed, use double quotes for table and column names
```

Minor nit and not necessarily worth changing for, but just FYI, you don't need `..`s on every line here; `.. ` starts a comment declaration block, so anything indented under it will be treated as a comment.

Reply:
> Minor nit and not necessarily worth changing for, but just FYI, you don't need `..`s on every line here; `.. ` starts a comment declaration block, so anything indented under it will be treated as a comment.

TIL!